### PR TITLE
send for approval notification

### PR DESCRIPTION
**Pull request by vikasrinvi**
#411 

- notification is sent when `Send-for-approval` action is trigger

### CODE REVIEW

File: app/Mail/HR/SendForApproval.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        return $this->to('vikasmrnv@gmail.com')
```
Comment (rathorevaibhav) [inline app/Mail/HR/SendForApproval.php]: remove hard coded value

Comment (rathorevaibhav) [reply app/Mail/HR/SendForApproval.php]: also add the name.
```php
$this->to($supervisor->email, $supervisor->name)
```

File: app/Models/HR/ApplicationRound.php [hunk 2]
```diff
@@ -71,6 +73,7 @@ public function _update($attr)
             case 'send-for-approval':
                 $fillable['round_status'] = 'confirmed';
                 $application->sendForApproval($attr['send_for_approval_person']);
+                Mail::send(new SendForApproval());
```
Comment (rathorevaibhav) [inline app/Models/HR/ApplicationRound.php]: 👍

**Comment by vikasrinvi** [comment]
@Satendra-SR Please have a Look on this PR and Provide the variable name accordingly

**Pull request merged**